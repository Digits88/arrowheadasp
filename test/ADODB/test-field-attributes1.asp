<!--#include file="../config/ADODB.inc"-->
Here is the test of Field Attributes.<br>
<%
    Dim CN, RS

    Set CN = Server.CreateObject("ADODB.Connection")
    CN.Open Connect_URL, Connect_Username, Connect_Password
    Set RS = CN.Execute("SELECT StringField, IntField FROM Test ORDER BY IntField")

    %>Attributes("StringField"): <%=RS("StringField").Attributes%><br>
    Attributes("IntField"): <%=RS("IntField").Attributes%><br>
<%

    CN.Close
%>

The End.
