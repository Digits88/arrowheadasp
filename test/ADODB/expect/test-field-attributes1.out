
Here is the test of Field Attributes.<br>
Attributes("StringField"): 8<br>
    Attributes("IntField"): 24<br>


The End.
